src/framework/components/light/light_data.js

```diff
@@ -14,6 +14,7 @@ pc.extend(pc, function () {
         this.innerConeAngle = 40;
         this.outerConeAngle = 45;
         this.falloffMode = pc.LIGHTFALLOFF_LINEAR;
+        this.mask = 1;
```

Comment:
As I may have mentioned before, I would like pc.Light and pc.Camera to be merged into the Light and Camera components at some point soon. Putting 'mask' into the LightComponent is like you've started doing this already. The current convention is that all light specific properties go in pc.Light. So it's not wrong, but it doesn't conform to the current 'architecture'. ;)